lizard_viewer.py

```diff
@@ -236,21 +251,28 @@ def show_data(self):
         self.dockwidget.set_all_status_bars_text(
             "{} downloaded.".format(asset_type.capitalize()))
 
+    def show_login_dialog(self):
+        """Function to show the login dialog."""
+        # Check whether the user is logged in
+        if self.username is None:
+            # The user_name_input QLineEdit becomes the active part of the
+            # login dialog
+            self.login_dialog.user_name_input.setFocus()
+            # Show the dialog
+            self.login_dialog.show()
+        else:
+            self.log_out()
+
     def log_in(self):
-        """Handle the log in."""
-        self.username = None
-        # Reset the Data type combobox
-        self.dockwidget.reset_datatypes_combobox()
-        # Remove the organisations from the combobox
-        self.dockwidget.remove_organisation_options()
+        """Function to handle logging in."""
         # Get the username
-        self.username = self.dockwidget.user_name_input.text()
+        self.username = self.login_dialog.user_name_input.text()
         # Check if the user exists
         try:
             # Get the possible users of the API the user has access to
             users = lizard_connector.connector.Endpoint(
                 username=self.username,
-                password=self.dockwidget.user_password_input.text(),
+                password=self.login_dialog.user_password_input.text(),
                 endpoint="users")
             users_ = users.download()
```

Comment:
Omdat je `users_` niet gebruikt kan je die dan laten zitten. Verder vind ik users versus users_ niet een heel duidelijk onderscheid in namen.